### PR TITLE
Fix remote profile example in about_Profiles.md.

### PR DESCRIPTION
**Pull request by sethvs**
Version(s) of document impacted
------------------------------
- [x] Impacts 6.1 document
- [x] Impacts 6.0 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

### CODE REVIEW

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_profiles.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutProfiles/reference/3.0/Microsoft.PowerShell.Core/About/about_profiles.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Core/About/about_profiles.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutProfiles/reference/4.0/Microsoft.PowerShell.Core/About/about_profiles.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Core/About/about_profiles.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutProfiles/reference/5.0/Microsoft.PowerShell.Core/About/about_profiles.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Core/About/about_Profiles.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutProfiles/reference/5.1/Microsoft.PowerShell.Core/About/about_Profiles.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Core/About/about_Profiles.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutProfiles/reference/6/Microsoft.PowerShell.Core/About/about_Profiles.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C19%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806190825308878-2596%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=P%2FGiL8m%2FGH6x%2FJQimxs%2B9jS9stso1jTM2HI52Uc38KE%3D&st=2018-06-19T08%3A30%3A39Z&se=2018-07-20T08%3A35%3A39Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by sdwheeler** [comment]
The change you made does not work either. We need a better example.

File: reference/3.0/Microsoft.PowerShell.Core/About/about_profiles.md [hunk 1]
```diff
@@ -314,13 +314,12 @@ Invoke-Command -Session $s -FilePath $profile
...
+populated, the command uses the explicit path to the profile. We use dot
+sourcing operator so that profile executes in the current scope on the
+remote computer and not in its own scope.
 
 ```powershell
-Invoke-Command -SessionName $s -ScriptBlock {
-$path = "$home\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1"
-Invoke-Command -FilePath $path
-}
+Invoke-Command -Session $s -ScriptBlock {. "$home\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1"}
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Core/About/about_profiles.md]: Change `$home` to `$env:USERPROFILE`.

**Comment by sdwheeler** [comment]
@sethvs OK, I was able to get this working except for the use of `$home`. Use `$env:USERPROFILE` instead.

**Comment by sethvs** [comment]
Should we change all the files for consistency or just the 3.0 and 4.0 versions?

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_profiles.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutProfiles/reference/3.0/Microsoft.PowerShell.Core/About/about_profiles.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Core/About/about_profiles.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutProfiles/reference/4.0/Microsoft.PowerShell.Core/About/about_profiles.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Core/About/about_profiles.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutProfiles/reference/5.0/Microsoft.PowerShell.Core/About/about_profiles.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Core/About/about_Profiles.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutProfiles/reference/5.1/Microsoft.PowerShell.Core/About/about_Profiles.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Core/About/about_Profiles.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutProfiles/reference/6/Microsoft.PowerShell.Core/About/about_Profiles.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C21%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806211137357209-2596%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Amt0ItTvfo6H7w4lRNpMmmxSLR4wJiha8635tp4%2FXwY%3D&st=2018-06-21T11%3A42%3A50Z&se=2018-07-22T11%3A47%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by sdwheeler** [comment]
@sethvs Yes, please change it for all versions. Thanks.

File: reference/6/Microsoft.PowerShell.Core/About/about_Profiles.md [hunk 2]
```diff
@@ -314,13 +314,12 @@ Invoke-Command -Session $s -FilePath $profile
...
+populated, the command uses the explicit path to the profile. We use dot
+sourcing operator so that profile executes in the current scope on the
+remote computer and not in its own scope.
 
 ```powershell
-Invoke-Command -SessionName $s -ScriptBlock {
-$path = "$home\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1"
-Invoke-Command -FilePath $path
-}
+Invoke-Command -Session $s -ScriptBlock {. "$home\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1"}
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Core/About/about_Profiles.md]: Change $home to $env:USERPROFILE

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Profiles.md [hunk 3]
```diff
@@ -314,13 +314,12 @@ Invoke-Command -Session $s -FilePath $profile
...
+populated, the command uses the explicit path to the profile. We use dot
+sourcing operator so that profile executes in the current scope on the
+remote computer and not in its own scope.
 
 ```powershell
-Invoke-Command -SessionName $s -ScriptBlock {
-$path = "$home\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1"
-Invoke-Command -FilePath $path
-}
+Invoke-Command -Session $s -ScriptBlock {. "$home\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1"}
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Profiles.md]: Change $home to $env:USERPROFILE

File: reference/5.0/Microsoft.PowerShell.Core/About/about_profiles.md [hunk 4]
```diff
@@ -314,13 +314,12 @@ Invoke-Command -Session $s -FilePath $profile
...
+populated, the command uses the explicit path to the profile. We use dot
+sourcing operator so that profile executes in the current scope on the
+remote computer and not in its own scope.
 
 ```powershell
-Invoke-Command -SessionName $s -ScriptBlock {
-$path = "$home\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1"
-Invoke-Command -FilePath $path
-}
+Invoke-Command -Session $s -ScriptBlock {. "$home\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1"}
```
Comment (sdwheeler) [inline reference/5.0/Microsoft.PowerShell.Core/About/about_profiles.md]: Change $home to $env:USERPROFILE

**Comment by sethvs** [comment]
Done.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Core/About/about_profiles.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutProfiles/reference/3.0/Microsoft.PowerShell.Core/About/about_profiles.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Core/About/about_profiles.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutProfiles/reference/4.0/Microsoft.PowerShell.Core/About/about_profiles.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Core/About/about_profiles.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutProfiles/reference/5.0/Microsoft.PowerShell.Core/About/about_profiles.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Core/About/about_Profiles.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutProfiles/reference/5.1/Microsoft.PowerShell.Core/About/about_Profiles.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Core/About/about_Profiles.md](https://github.com/sethvs/PowerShell-Docs/blob/aboutProfiles/reference/6/Microsoft.PowerShell.Core/About/about_Profiles.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C21%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201806211618242840-2596%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Rk2S8%2FFppSrYkuRQrxbNYh5M2mnr5CLZGDcIyYV%2Fe8A%3D&st=2018-06-21T16%3A24%3A41Z&se=2018-07-22T16%3A29%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**